src/store.jsx

```diff
@@ -6,9 +6,37 @@ import reducers from "./ducks";
 
 import localStorageMiddleware from "./middlewares/localStorageMiddleware";
 
+const getCookie = (cookieName) => {
```

Comment:
I think this function is very usefull and could be used anywhere in the project, so it makes sense to move it to utils